pkg/storage/export/export/export.go

```diff
@@ -839,7 +853,7 @@ func (ctrl *VMExportController) createExporterPodManifest(vmExport *exportv1.Vir
 		Value: "/token/token",
 	}, corev1.EnvVar{
 		Name:  "DEADLINE",
-		Value: currentTime().Add(deadline).Format(time.RFC3339),
+		Value: getDeadlineValue(deadline, vmExport).Format(time.RFC3339),
```

Comment:
I don't think you need to change the current behavior because of the following:
If the TTL < Deadline, then the TTL will cause the VMExport to be deleted, which in turn will delete the exporter pod. If the TTL > Deadline, we have the current behavior. With this change it is possible the pod gets destroyed and recreated slightly before the VMExport is deleted, and the new pod gets deleted anyway. So I think it simpler and more correct to just keep the deadline as is.

Reply:
I don't think we get a reconcile triggered at TTL o'clock this way

Reply:
So you are using the deadline expiring on the pod to trigger a reconcile on the VMExport? I am trying to think of a better and reliable way to trigger a reconcile when the TTL expires, and I can't.

Reply:
This was actually an idea from @mhenriks (thx)
it was either that or requeues